in_text/text_02_drivers.yml

```
@@ -457,9 +457,9 @@ process-description: >-
   Stream observations and driver data were combined from multiple sources to achieve the most complete temporal coverage possible
   from 1980-2022. Historical weather driver data from NLDAS, GridMET, and NOAA GEFS (in this release and Oliver et al. 2022) were used to
   train the models, while forecasted weather data (NOAA GEFS) was used to evaluate real-time forecasts. Gridded weather
-  drivers were were aggregated to our reservoir polygons
-  and stream polylines either using grid-to-point intersection or weighted averages of grids that intersected
-  a stream reach's hydrologic response units (HRUs).
+  drivers were aggregated to the reservoir polygons
+  and stream polylines included in this release either using grid-to-point intersection or weighted averages of grids that intersected
+  a stream reach's hydrologic response units (HRUs). 
```

where does this data come from? probably  need a reference here
```suggestion
  a stream reach's hydrologic response units (HRUs).
```